pkg/controller/petset/pet.go

```diff
@@ -103,8 +111,9 @@ func (p *petSyncer) Sync(pet *pcb) error {
 		return p.Update(realPet, pet)
 	}
 	if p.blockingPet != nil {
-		glog.Infof("Create of %v in PetSet %v blocked by unhealthy pet %v", pet.pod.Name, pet.parent.Name, p.blockingPet.pod.Name)
-		return nil
+		message := fmt.Sprintf("Create of %v in PetSet %v blocked by unhealthy pet %v", pet.pod.Name, pet.parent.Name, p.blockingPet.pod.Name)
+		glog.Error(message)
```

Comment:
this is actually not an error, it will happen in the normal case. Please Infof. 

For some context, the point of this unhelthy pet blocking is because we don't want to create pet-1 before pet-0 is healthy (init + running + ready probe), but we're consulting a cache to check the state of pet-0, and this cache won't even have pet-0 just after we've created it. So we cache the last created pet in a member var and it blocks all subsequent creates (but not subsequent syncs/updates). Whenever the watch delivers pet-0 as ready/running, we clear the blockign pet and pet-1 can be created (pet-1 then becomes blocking pet...).